src/sentry/static/sentry/app/components/activity/item.jsx

```diff
@@ -51,133 +51,129 @@ const ActivityItem = React.createClass({
     let orgId = this.props.orgId;
     let project = item.project;
     let issue = item.issue;
+    let issueLink = issue
+      ? <Link to={`/${orgId}/${project.slug}/issues/${issue.id}/`}>{issue.shortId}</Link>
+      : null;
 
     switch (item.type) {
       case 'note':
-        return tct('[author] commented on [link:an issue]', {
+        return tct('[author] commented on [issue]', {
           author: author,
-          link: (
-            <Link
-              to={`/${orgId}/${project.slug}/issues/${issue.id}/activity/#event_${item.id}`}
```

Comment:
looks like you're actually changing the link here--not sure if that was intentional?